testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -175,20 +177,59 @@ def uma_metadata(self, name, endpoint, credentials, **common_features):
 class Responses(Section):
     """Section which contains response configuration"""
 
-    def add_simple(self, auth_json, name="simple", key="data", **common_features):
-        """Adds simple response to AuthConfig"""
-        self.add(
+    def _add(
+        self,
+        name: str,
+        value: dict,
+        wrapper_key: str = None,
+        wrapper: Literal["httpHeader", "envoyDynamicMetadata"] = None,
+        **common_features
+    ):
+        """Add response to AuthConfig"""
+        value["wrapper"] = wrapper
+        value["wrapperKey"] = wrapper_key
```

Comment:
is this supposed to be or safe to set it everywhere? Every object will have it set to None (at least). I see that this is the purpose of this function, anyway it looks odd.

Reply:
I would rather see the if, just to be sure